src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        final int figureNumbers = 6;
+
+        Figure[] figures = new Figure[figureNumbers];
+
+        for (int i = 0; i < figures.length; i++) {
+            if (i >= figureNumbers / 2) {
+                figures[i] = figureSupplier.getDefaultFigure();
+            } else {
+                figures[i] = figureSupplier.getRandomFigure();
+            }
+            System.out.println(figures[i].toString());
```

Comment:
```suggestion
            figures[i].draw();
```
Task is not implemented. Add one more interface with draw() method.
![image](https://user-images.githubusercontent.com/20326858/197403455-1733db05-35b9-4c67-983c-8943e126a0c1.png)
![image](https://user-images.githubusercontent.com/20326858/197403481-7000e412-ce6c-4e62-9142-8ccd2157201d.png)